lib/workflows/load.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+const dir = require('node-dir');
+const yaml = require('js-yaml');
+const kebab = require('lodash/kebabCase');
+const globalConfig = require('config');
+const path = require('path');
+
+/**
+ * Gets all Workflows
+ *
+ * @returns {Promise} - An {Array} containing {Object} for each workflow, including `name` {String}, `id` name {String}, and steps {Object} loaded from YAML config file
+ */
+const load = () => {
+  const workflows = [];
+  const ids = [];
+  let workflowsPath = path.join(process.cwd(), 'workflows');
+
+  if (globalConfig.hasOwnProperty('workflows')) {
+    if (globalConfig.content.hasOwnProperty('workflows')) {
```

Comment:
It should be fairly trivial to make workflows pluggable at this point, why not just do it? Then we can distribute self-publish and editor-approve immediately